README.md

```diff
@@ -265,6 +266,54 @@ other operations with data.
       corva.scheduled(my_app, event)
    ```
 
+## Logging
+As apps are executed very frequently
+(once a second or so),
+unlimited logging can lead to huge amounts of data.
+
+The SDK provides a `Logger` object,
+which is a safe way for logging in apps.
+
+The `Logger` is a `logging.Logger` instance
+and should be used like every other Python logger.
+
+The `Logger` has following features:
+1. Log messages are injected with contextual information,
+   which makes it easy to filter through logs
+   while debugging issues.
+1. Log message length is limited. 
+   Too long messages are truncated to not exceed the limit. 
+   Set by `LOG_THRESHOLD_MESSAGE_SIZE` env variable. 
+   Default value is `1000` symbols or bytes.
+2. Number of log messages is limited.
+   After reaching the limit logging gets disabled.
+   Set by `LOG_THRESHOLD_MESSAGE_COUNT` env variable.
+   Default value is `15`.
+3. Logging level can be set using `LOG_LEVEL` env variable.
+   Default value is `WARN`.
```

Comment:
Should default level be INFO? What do you think?

Reply:
Node SDK uses lowercase characters to set log level. Is it possible to make these the same? @katsanva

Reply:
Either node or python needs to implement some changes/mapping so that we will be on the same page.
I think this should be done in `python-sdk` because we have some amount of apps written and configured in node already
The supported list of log levels are `'fatal', 'error', 'warn', 'info', 'debug'`

Reply:
This is where python and nodejs differ. e.g there is no `fatal` in python. I think we should leave log levels as expected by users of corresponding languages, so there is less learning

Reply:
@oleksii-symon-corva-ai disagree. I think we need to provide interchangeable abilities in SDK so that we can apply the language-agnostic configuration to the apps

Reply:
Users define the these env variables using manifest.json file.

Reply:
IMO it's much easier to explain to the user that manifest.json can contain
```
LOG_LEVEL is used to configure log level, possible values are 'fatal', 'error', 'warn', 'info', 'debug'
``` 

vs 

```
LOG_LEVEL is used to configure log level. 
If you are using nodejs possible values are 'fatal', 'error', 'warn', 'info', 'debug'. 
If you are going to use Python please use 'DEBUG', 'INFO', 'WARN', 'ERROR'
```